src/core/statement.cpp

```diff
@@ -682,28 +644,42 @@ void statement_impl::describe()
         switch (dtype)
         {
         case dt_string:
-            bind_into<dt_string>();
-            break;
         case dt_blob:
-            bind_into<dt_string>();
-            break;
         case dt_xml:
-            bind_into<dt_string>();
+        {
```

Comment:
This is pretty minor, but the indentation here is weird, `break` should normally be aligned with the `}`, i.e. I'd indent this block (and all the similar ones below).